data/schema/core/addons.cfg

```diff
@@ -0,0 +1,470 @@
+	{SIMPLE_KEY force_lock_settings bool}
+	[tag]
+		name="story"
+		# TODO: Remove this once the actual supported tags are properly documented (basically messages, insert_tag, if/then/else, switch/case)
+		any_tag=yes
+		# TODO: Is this really recognized at story toplevel? Wiki claims it is.
+		[tag]
+			name="image"
+			max=infinite
+			{SIMPLE_KEY x int}
+			{SIMPLE_KEY y int}
+			{SIMPLE_KEY centered bool}
+			{SIMPLE_KEY file string}
+			{SIMPLE_KEY delay int}
+			# These keys possibly not documented?
+			{SIMPLE_KEY scale_vertically bool}
+			{SIMPLE_KEY scale_horizontally bool}
+			{SIMPLE_KEY scale bool}
+			{SIMPLE_KEY tile_vertically bool}
+			{SIMPLE_KEY tile_horizontally bool}
+			{SIMPLE_KEY tile bool}
+		[/tag]
+		[tag]
+			name="part"
+			max=infinite
+			{SIMPLE_KEY background string}
+			{DEFAULT_KEY scale_background bool yes}
+			{SIMPLE_KEY story t_string}
+			{SIMPLE_KEY show_title bool}
+			{SIMPLE_KEY title t_string}
+			{SIMPLE_KEY music string}
+			{SIMPLE_KEY sound string_list}
+			{DEFAULT_KEY text_layout string bottom}
+			{DEFAULT_KEY title_alignment text_alignment left}
+			{SIMPLE_KEY delay int} # Not documented?
+			[tag]
+				name="background_layer"
+				max=infinite
+				{SIMPLE_KEY image string}
+				{SIMPLE_KEY scale_vertically bool}
+				{SIMPLE_KEY scale_horizontally bool}
+				{SIMPLE_KEY scale bool}
+				{SIMPLE_KEY keep_aspect_ratio bool}
+				{SIMPLE_KEY tile_vertically bool}
+				{SIMPLE_KEY tile_horizontally bool}
+				{SIMPLE_KEY tile bool}
+				{SIMPLE_KEY base_layer bool}
+			[/tag]
+			[tag]
+				name="if"
+				max=infinite
+				super="$conditional_wml"
+				[tag]
+					name="then"
+					super="scenario/story/part"
+				[/tag]
+				[tag]
+					name="else"
+					super="scenario/story/part"
+				[/tag]
+			[/tag]
+			# TODO: Switch/case
+			[link]
+				name="scenario/story/image"
+			[/link]
+		[/tag]
+	[/tag]
+	[tag]
+		name="label"
+		max=infinite
+		{SIMPLE_KEY x int}
+		{SIMPLE_KEY y int}
+		{SIMPLE_KEY text t_string}
+		{SIMPLE_KEY tooltip t_string} # Is this documented?
+		{SIMPLE_KEY immutable bool}
+		{SIMPLE_KEY visible_in_fog bool}
+		{SIMPLE_KEY visible_in_shroud bool}
+		{SIMPLE_KEY team_name string}
+	[/tag]
+	[tag]
+		name="item"
+		max="infinite"
+		{SIMPLE_KEY x range_list}
+		{SIMPLE_KEY y range_list}
+		{SIMPLE_KEY image string}
+		{SIMPLE_KEY halo string}
+	[/tag]
+	[tag]
+		name="time"
+		max=infinite
+		{REQUIRED_KEY id string}
+		{SIMPLE_KEY name t_string}
+		{SIMPLE_KEY image string}
+		{SIMPLE_KEY mask string}
+		{SIMPLE_KEY lawful_bonus int}
+		{SIMPLE_KEY red int}
+		{SIMPLE_KEY green int}
+		{SIMPLE_KEY blue int}
+		{SIMPLE_KEY sound string}
+	[/tag]
+	[tag]
+		name="time_area"
+		max=infinite
+		{SIMPLE_KEY x range_list}
+		{SIMPLE_KEY y range_list}
+		[link]
+			name="scenario/time"
+		[/link]
+	[/tag]
+	[tag]
+		name="side"
+		min=1
+		max=infinite
+		super="$side"
+		{REQUIRED_KEY side int}
+		# TODO: Pretty sure this isn't actually required?
```

Comment:
technicially not, but it has a different default value in sp and mp, so if you want your campaign to work in both in sp and mp you should specify it, (in particular since iirc 1.13 allow syou to start sp campaign in the mp create dialog). We shodul really make this key manditory.

Reply:
I think making it mandatory might be a bit much…